tron/mcp.py

```diff
@@ -79,23 +96,55 @@ def _load_config(self, reconfigure=False, namespace_to_reconfigure=None):
                 namespace_to_reconfigure=namespace_to_reconfigure,
             )
 
+    def _update_metrics(self):
```

Comment:
```suggestion
    def _update_metrics(self) -> None:
```
non-blocking nit since i'm trying to add types to all of tron :p